data.yml

```diff
@@ -2030,7 +2408,7 @@ components:
           items:
             $ref: '#/components/schemas/AccountResponseResource'
         links:
-          $ref: '#/components/schemas/ResourceLink'
+          $ref: '#/components/schemas/ResourceaccountLink'
```

Comment:
Should probably capitalise that A in `account`

Reply:
@basiq-chris Nice pickup, here you go, i have fixed it [here](https://github.com/basiqio/api-ref/pull/331/commits/45b5a5ea774036f2d48e9742aff5f3d79a19abb0).